Packs/FeedElasticsearch/Integrations/FeedElasticsearch/FeedElasticsearch.yml

```diff
@@ -100,7 +105,7 @@ configuration:
   name: feedExpirationInterval
   type: 1
   required: false
-- defaultvalue: '240'
+- defaultvalue: '10'
```

Comment:
what was this changed?

Reply:
This is the fetch interval.
SInce in the previous version it fetched all the indicators in one batch, it was making sense to set the feed interval for 4 hours.
In the current version it will fetch in batches, so it doesn't really make sense to wait 4 hours until the next batch.